Resources/Textures/_RMC14/Structures/campfire.rsi/meta.json

```
@@ -0,0 +1,26 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "https://github.com/cmss13-devs/cmss13/blob/7cb618c69b75873f3ce893022fe08d1233b3152d/icons/obj/structures/structures.dmi",
+  "size": {
+    "x": 32,
+    "y": 32
+  },
+  "states": [
+    {
+      "name": "campfire"
+    },
+    {
+    "name": "burning",
+    "delays": [
+        [
+    0.3,
+    0.3,
+    0.3,
+    0.3,
+    0.3
+    ]
+    ]
+    }
+  ]
+}
```

I am baffled at how cooked the indentation got here
```suggestion
{
  "version": 1,
  "license": "CC-BY-SA-3.0",
  "copyright": "https://github.com/cmss13-devs/cmss13/blob/7cb618c69b75873f3ce893022fe08d1233b3152d/icons/obj/structures/structures.dmi",
  "size": {
    "x": 32,
    "y": 32
  },
  "states": [
    {
      "name": "campfire"
    },
    {
      "name": "burning",
      "delays": [
        [
          0.3,
          0.3,
          0.3,
          0.3,
          0.3
        ]
      ]
    }
  ]
}
```

---

no idea what happened here